source/reference/minio-server/settings/notifications/mysql.rst

```diff
@@ -40,15 +40,15 @@ The following commands set two distinct MySQL service endpoints as ``PRIMARY`` a
       .. code-block:: shell
          :class: copyable
       
-         set MINIO_NOTIFY_MYSQL_ENABLE_PRIMARY="on"
-         set MINIO_NOTIFY_MYSQL_DSN_STRING_PRIMARY="username:password@tcp(mysql.example.com:3306)/miniodb"
-         set MINIO_NOTIFY_MYSQL_TABLE_PRIMARY="minioevents"
-         set MINIO_NOTIFY_MYSQL_FORMAT_PRIMARY="namespace"
+         export MINIO_NOTIFY_MYSQL_ENABLE_PRIMARY="on"
```

Comment:
Is there a `windows` | `not windows` differentiator for this text?

Reply:
no, we don't produce windows docs for the Minio Server reference docs. They're linux only.